openedx/features/custom_fields/multiselect_with_other/forms/fields.py

```diff
@@ -1,3 +1,5 @@
+""" This module contains formfield for to select multiple options from given choices"""
```

Comment:
*This module contains **formfield to select** multiple options from given choices

Reply:
Please change it to 
```
FormField for multi select with other option
```